app/controllers/community_resources_controller.rb

```diff
@@ -0,0 +1,37 @@
+class CommunityResourcesController < ApplicationController
+  before_filter :set_resource, :except => [:index, :create]
```

Comment:
@gschorkopf found that in a code sample online. Will update to the newer syntax.

On a side note - do you know any good online tutorials around creating an API with Rails? I found a fair amount of stuff but most of it was older (2014) and wasn't sure if they used current best practices.